admin/tools/docker/tagged/Dockerfile.tpl

```diff
@@ -3,16 +3,17 @@ MAINTAINER Fabrice Jammes <fabrice.jammes@in2p3.fr>
 
 USER root
 
-RUN apt-get update && apt-get -y install byobu git vim gdb lsof
+RUN apt-get update
+RUN apt-get -y install byobu git vim gdb lsof
 
 USER qserv 
 
 WORKDIR /home/qserv
 
-RUN mkdir src && cd src && git clone -b {{GIT_TAG_OPT}} --single-branch https://github.com/LSST/qserv
-WORKDIR /home/qserv/src/qserv
-RUN bash -c ". /qserv/stack/loadLSST.bash && setup -r . -t qserv-dev && eupspkg -er build"
+RUN mkdir src
```

Comment:
Good point, I'll do it in one line, please note that git clone can create src/qserv.